### DISCUSSION TITLE
DNS entry to access web-gui with logical name

### DISCUSSION THREAD

**Discussion by FIAV1** [RaspAP/raspap-webgui#857] [Q&A]
Is it possible to add a DNS entry to access webgui in AP-STA mode using a logical name instead of ip address?

**Comment by billz** [comment]
The `avahi-daemon` gives you this facility via multicast DNS (mDNS). This is included in the RPi OS kernel and on computers with the Bonjour service installed (all macOS machines and Windows PCs with Apple iTunes). This means you can access the web UI in the browser with http://raspberrypi.local/, assuming a default hostname.

Check its status with `sudo systemctl status avahi-daemon.service `.

**Reply by FIAV1** [reply]
Does it work even when I'm connected directly to the Raspberry?

**Reply by FIAV1** [reply]
Updates: if i ping raspberrypi from inside the raspberry i get response; instead, if i ping raspberrypi from my pc connected to the raspberry, I get `Temporary failure in name resolution`. Are there some peculiar configurations to be done?

**Reply by billz** [reply]
Your PC will need the Bonjour service.
https://docs.raspap.com/faq/#access

**Reply by FIAV1** [reply]
Already has it, I can reach http://raspberrypi.local from my pc when they're both connected to my home network. This doesn't work when my pc is connected directly to the raspberry.
